### PR TITLE
Make building with Bullet on by default

### PR DESCRIPTION
**Pull request by jonathan-owens**

### CODE REVIEW

**Comment by RussTedrake** [comment]
the failures on appveyor are the ones I expected / remembered.  it's not building bullet that was failing, it's linking to bullet from drake.

**Comment by jonathan-owens** [comment]
See: https://github.com/RobotLocomotion/bullet-pod/pull/9

**Comment by RussTedrake** [comment]
please update the sha and hopefully we're in business.  we have someone that will use it immediately!  thanks.

**Comment by RussTedrake** [comment]
fantastic.  this is great to have.  thanks.

File: CMakeLists.txt [hunk 1]
```diff
@@ -34,13 +34,13 @@ option(REMOVE_UNUSED_EXTERNALS "enable this to remove those projects from disk"
 # ON by default:
 option(WITH_EIGEN		"required c++ matrix library.  only disable if you have it already."    ON)
 option(WITH_SWIGMAKE    "helper tools to build python & MATLAB wrappers for C++ libraries with Eigen" ON)
+option(WITH_BULLET 		"used for collision detection" 	ON) # almost works.  just one remaining link error..?
```
Comment (RussTedrake) [inline CMakeLists.txt]: ah.  i guess it would have been better to zap that trailing comment.

**Comment by jonathan-owens** [comment]
Ah bummer -- want me to make another merge request with that comment deleted?

**Comment by RussTedrake** [comment]
Yes.  Thanks

**Pull request merged**